src/lib/ketrew_configuration.mli

```diff
@@ -180,3 +142,47 @@ val connection: client -> string
 val token: client -> string
 
 val standalone_of_server: server -> standalone
+
+
+val load_exn:
+  ?and_apply:bool ->
+  ?profile:string ->
+  [ `From_path of string
+  | `Guess
+  | `In_directory of string
+  | `Override of t ] ->
+  t
+(** Load a configuration.
```

Comment:
ok for putting the optional arguments after, will do.

I don't understand the second point (?)

putting ``` `Override c``` outside of `load_exn` would mean more work for the callers of the function (every time have to figure out what to do). How would you organize that?